haiku-generator-rs/spin.toml

```diff
@@ -0,0 +1,18 @@
+spin_manifest_version = "1"
+authors = ["tpmccallum <tim.mccallum@fermyon.com>"]
+description = "An application that can write haiku poetry via Fermyon Serverless AI"
+name = "haiku-generator-rs"
+trigger = { type = "http", base = "/" }
+version = "0.1.0"
+
+[[component]]
+id = "haiku-generator-rs"
+source = "target/wasm32-wasi/release/haiku_generator_rs.wasm"
+allowed_http_hosts = []
+ai_models = ["llama2-chat"]
```

Comment:
The readme talks about downing a `gpt2` based model while it mentions `llama2` here.

Reply:
Thanks @karthik2804 
It would be great to get this approved if you have a spare minute; we link to this Rust application in the blog post, which will be published today.